back/backend/lib/formatters.py

```diff
@@ -0,0 +1,122 @@
+def parse_time(time):
+    centiseconds = time % 100
+    res = time // 100
+    seconds = res % 60
+    res = res // 60
+    minutes = res % 60
+    hours = res // 60
+    return (hours, minutes, seconds, centiseconds)
+
+
+def FormatStandard(time, trim_zeros):
```

Comment:
Function names in Python should be lower case and snake case, e.g., `format_standard`